src/Arcus.Messaging.Tests.Integration/Fixture/ServiceBusQueueAndTopicProgram.cs

```diff
@@ -45,7 +45,7 @@ public static void main(string[] args)
                     });
 
                     services.AddServiceBusQueueMessagePump(configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING_WITH_TOPIC"])
-                            .AddServiceBusTopicMessagePump("Receive-All", configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING_WITH_QUEUE"])
+                            .AddServiceBusTopicMessagePump("Test-Receive-All-Topic-And-Queue", configuration => configuration["ARCUS_SERVICEBUS_CONNECTIONSTRING_WITH_QUEUE"])
```

Comment:
I presume it's on purpose that you pass queue connection info for topic?

Reply:
Hmm, no 😄 .